pkg/localmetrics/localmetrics.go

```diff
@@ -23,54 +23,66 @@ import (
 	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
 )
 
+const operatorName = "aws-account-operator"
+
 var log = logf.Log.WithName("localmetrics")
 
 var (
-	MetricTotalAWSAccounts = prometheus.NewGaugeVec(prometheus.GaugeOpts{
-		Name: "aws_account_operator_total_aws_accounts",
-		Help: "Report how many accounts have been created in AWS org",
-	}, []string{"name"})
-	MetricTotalAccountCRs = prometheus.NewGaugeVec(prometheus.GaugeOpts{
-		Name: "aws_account_operator_total_account_crs",
-		Help: "Report how many account CRs have been created",
-	}, []string{"name"})
-	MetricTotalAccountCRsUnclaimed = prometheus.NewGaugeVec(prometheus.GaugeOpts{
-		Name: "aws_account_operator_total_accounts_crs_unclaimed",
-		Help: "Report how many account CRs are unclaimed",
-	}, []string{"name"})
-	MetricTotalAccountCRsClaimed = prometheus.NewGaugeVec(prometheus.GaugeOpts{
-		Name: "aws_account_operator_total_accounts_crs_claimed",
-		Help: "Report how many account CRs are claimed",
-	}, []string{"name"})
-	MetricTotalAccountCRsFailed = prometheus.NewGaugeVec(prometheus.GaugeOpts{
-		Name: "aws_account_operator_total_accounts_crs_failed",
-		Help: "Report how many account CRs are failed",
-	}, []string{"name"})
-	MetricTotalAccountClaimCRs = prometheus.NewGaugeVec(prometheus.GaugeOpts{
-		Name: "aws_account_operator_total_aws_account_claim_crs",
-		Help: "Report how many account claim CRs have been created",
-	}, []string{"name"})
-	MetricTotalAccountCRsReady = prometheus.NewGaugeVec(prometheus.GaugeOpts{
-		Name: "aws_account_operator_total_aws_accounts_crs_ready",
-		Help: "Report how many account CRs are ready",
-	}, []string{"name"})
-	MetricPoolSizeVsUnclaimed = prometheus.NewGaugeVec(prometheus.GaugeOpts{
-		Name: "aws_account_operator_pool_size_vs_unclaimed",
-		Help: "Report the difference between the pool size and the number of unclaimed account CRs",
-	}, []string{"name"})
-	MetricTotalAccountPendingVerification = prometheus.NewGaugeVec(prometheus.GaugeOpts{
-		Name: "aws_account_operator_total_aws_account_pending_verification",
-		Help: "Report the number of accounts waiting for enterprise support and EC2 limit increases in AWS",
-	}, []string{"name"})
+	MetricTotalAWSAccounts = prometheus.NewGauge(prometheus.GaugeOpts{
```

Comment:
Would it make sense to abstract these to a function?

```
func newGauge(name string, help string) prometheus.Gauge { #or whatever the type would be
    return prometheus.NewGauge(prometheus.GaugeOpts{
        Name: name,
        Help: help,
        ConstLabels: prometheus.Labels{"name": operatorName},
    })
}
```

Reply:
Thanks for your review. IMO a function is not necessary since the current way is already a common pattern.